angular.json

```diff
@@ -131,7 +135,7 @@
           "builder": "@angular-devkit/build-angular:ng-packagr",
           "options": {
             "tsConfig": "projects/atft/tsconfig.lib.json",
-            "project": "projects/atft/ng-package.json"
+            "project": "projects/atft/ng-package.json",
```

Comment:
trailing comma causing storybook failure (found comment in stackoverflow)

Reply:
thanks, will try to catch this in the IDE before fix it